forge/src/hooks/RequestHook.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: GPL-3.0-only
```

Comment:
This is good but I think we should get it into `tnt-core`, publish that, and export/re-import those contracts here before this is merged IMO. Wdyt?

Reply:
once we finalize the design of these hooks, before we merge into `main` branch we will move these contracts to tnt-core.